File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/emr/OCS2015U04LoadExaminationInfo.java
package nta.med.data.model.ihis.emr;

import java.util.Date;

public class OCS2015U04LoadExaminationInfo {
    private Double pkout1001;
    private Date naewonDate;
    private String gwa;
	public OCS2015U04LoadExaminationInfo(Double pkout1001, Date naewonDate,
			String gwa) {
		super();
		this.pkout1001 = pkout1001;
		this.naewonDate = naewonDate;
		this.gwa = gwa;
	}
	public Double getPkout1001() {
		return pkout1001;
	}
	public void setPkout1001(Double pkout1001) {
		this.pkout1001 = pkout1001;
	}
	public Date getNaewonDate() {
		return naewonDate;
	}
	public void setNaewonDate(Date naewonDate) {
		this.naewonDate = naewonDate;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
    
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/OcsEmr/OCS2015U04LoadExaminationInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.OcsEmr
{
    [Serializable]
	public class OCS2015U04LoadExaminationInfo
	{
		private String _pkout1001;
		private String _naewonDate;
		private String _gwa;

		public String Pkout1001
		{
			get { return this._pkout1001; }
			set { this._pkout1001 = value; }
		}

		public String NaewonDate
		{
			get { return this._naewonDate; }
			set { this._naewonDate = value; }
		}

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public OCS2015U04LoadExaminationInfo() { }

		public OCS2015U04LoadExaminationInfo(String pkout1001, String naewonDate, String gwa)
		{
			this._pkout1001 = pkout1001;
			this._naewonDate = naewonDate;
			this._gwa = gwa;
		}

	}
}